spec/dummy/.env

```
@@ -5,3 +5,5 @@ DATABASE_PASSWORD=passwd
 REDIS_URL=redis://redis:6379/1
 
 RAILS_PORT=8000
+
+SIDEKIQ_HOSTS=dcsiloifoodcatalogimporter_sidekiq_1;dcsiloifoodcatalogimporter_sidekiq_2
```

I think you can be more generic here instead of using the name of services.
```suggestion
SIDEKIQ_HOSTS=sidekiq_host_1;sidekiq_host_2
```